src/howitz/endpoints.py

```diff
@@ -113,34 +122,47 @@ def get_current_events():
     return table_events
 
 
-def poll_current_events():
-    try:
-        current_app.event_manager.get_events()
-    except NotConnectedError as notConnErr:
-        if session["not_connected_counter"] > 1:  # This error is not intermittent - increase counter and handle
-            current_app.logger.exception('Recurrent NotConnectedError %s', notConnErr)
-            session["not_connected_counter"] += 1
-            raise
-        else:  # This error is intermittent - increase counter and retry
-            current_app.logger.exception('Intermittent NotConnectedError %s', notConnErr)
-            session["not_connected_counter"] += 1
-            current_app.event_manager.get_events()
-            pass
+def update_events():
+    updated_ids = set()
 
-    events = current_app.event_manager.events
+    while True:
+        updated = current_app.updater.get_event_update()
+        if updated:
+            updated_ids.add(updated)
+            continue
+        if updated is False:
+            break
```

Comment:
While `is False` may work here, you are in fact making an *identity* comparison with the built-in object `False`, not a value comparison.  The called function may actually return the bool object `False`, but you shouldn't rely on it when making boolean comparisons.   `if not updated` would be the idiomatic way.

This whole section would read better as 
```suggestion
        if not updated:
            break
        updated_ids.add(updated)
```

Reply:
@podliashanyk other than the code being just two lines of simple iterators, no.

Reply:
I'd add that in 99% of cases you will ever need the `is` keyword in Python, it is when explicitly comparing a value to `None`, as in `x is None` or `x is not None`.